src/tactic/ui/panel/base_table_layout_wdg.py

```diff
@@ -1276,17 +1289,12 @@ def get_action_wdg(my):
             outer.add(my.view_save_dialog)
 
         outer.add_style("min-width: 750px")
-        #outer.add_style("width: 300px")
-        #outer.add_style("overflow: hidden")
-        #outer.add_class("spt_resizable")
-
-        #div.add_style("min-width: 800px")
         div.add_style("height: %s" % height)
         div.add_style("margin: 0px -1px 0px -1px")
 
         
         
-
```

Comment:
When you comment out visual effect, I suggest leaving a reason behind to prevent wondering why it is turned off. Is it for a particular browser or just not trendy any more?